### ISSUE TITLE
[bug] helm chart crds need to be updated

### ISSUE THREAD

**Issue by char-iot** [Keyfactor/command-cert-manager-issuer#9]
for example caSecretName is missing. endEntityName etc. is not in README.md guide.

https://github.com/Keyfactor/command-cert-manager-issuer/blob/09ff15488e1579410260d324ab3f3a98d2b78a92/deploy/charts/command-cert-manager-issuer/templates/crds/clusterissuers.yaml#L35-L58

**Comment by m8rmclaren** [comment]
Hello! Thank you for bringing this to my attention, this appears to be an inconsistency between the `README.md` and the CRD. I'll update the `README.md` with the correct value of `caBundleSecretName`.

**Comment by char-iot** [comment]
mhm, there might be another error in container image or somewhere else,
when I used caBundleSecretName, it kept complaining x509: certificate signed by unknown authority. but when I added this block to clusterissuer, it stopped.
```yaml
caSecretName:
  description: caSecretName.
  type: string
```

**Comment by char-iot** [comment]
also in example spec:
```yaml
spec:
  hostname: ""
  commandSecretName: ""
  certificateTemplate: ""
  certificateAuthorityLogicalName: ""
  certificateAuthorityHostname: ""
  caSecretName: ""
```
but in crd, it requires certificateAuthorityName , certificateProfileName and endEntityProfileName instead
https://github.com/Keyfactor/command-cert-manager-issuer/blob/09ff15488e1579410260d324ab3f3a98d2b78a92/deploy/charts/command-cert-manager-issuer/templates/crds/clusterissuers.yaml#L53-L58

**Comment by m8rmclaren** [comment]
My apologies, I understand the issue now; the CRDs used in the Helm chart are for the wrong external cert-manager issuer. I've addressed this in 5fb90a4bbf9686fb3b8e73a373cc82294251bf48 and will be implemented in v1.0.4 (#11). For the meantime, you're welcome to use the [manifest installation](https://github.com/Keyfactor/command-cert-manager-issuer#installation-from-manifests).

**Comment by char-iot** [comment]
no worries. thank you very much for this crd. been waiting for something like this for a while. I'll wait for helm chart release because our keyfactor test server is not ready anyway.
